build/docker/postgres/create_db.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 
-for db in account device mediaapi syncapi roomserver signingkeyserver keyserver federationsender appservice naffka; do
+for db in userapi_account userapi_device mediaapi syncapi roomserver signingkeyserver keyserver federationsender appservice naffka; do
```

Comment:
```suggestion
for db in userapi_accounts userapi_devices mediaapi syncapi roomserver signingkeyserver keyserver federationsender appservice naffka; do
```
https://github.com/matrix-org/dendrite/blob/master/dendrite-config.yaml#L318
https://github.com/matrix-org/dendrite/blob/master/dendrite-config.yaml#L323

Reply:
Probably should update https://github.com/matrix-org/dendrite/blob/master/build/docker/config/dendrite-config.yaml#L301 to be consistent.